app/views/declaration/additionaldeclarationtype/declaration_type.scala.html

```diff
@@ -19,11 +19,18 @@
 @import forms.declaration.additionaldeclarationtype.AdditionalDeclarationTypeStandardDec
 @import models.requests.JourneyRequest
 @import views.Title
+@import config.AppConfig
 
-@this(main_template: views.html.main_template)
+@this(main_template: views.html.main_template, appConfig: AppConfig)
```

Comment:
@PatrykRudnicki Is this required? I think we made a decision of not passing the appConfig explicitly into the views

Reply:
Yes, AppConfig is required, but it's pointless to add this in every file.

It should be a separate component for this element and this component can have appConfig injected. In that case we're avoiding adding appConfig to every file, less work, less broken tests and easier usage.

Reply:
Done